extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -618,10 +621,13 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
       return AD_STATE.PENDING;
     }
 
-    if (!this.isCurrentAdLoaded_ || pageBeforeAd.isAd() ||
-        pageAfterAd.isAd()) {
-      // if we are going to cause two consecutive ads or ad is still
-      // loading we will try again on next user interaction
+    if (!this.isCurrentAdLoaded_ || this.adsAreBlocked_(pageBeforeAd) ||
+        pageBeforeAd.isAd() || pageAfterAd.isAd()) {
+      // There are three checks here that we check before inserting an ad. If
+      // any of these fail we will try again on next page navigation.
+      // 1. Ad must be loaded.
+      // 2. Pubs can opt out of ad placement using 'block-ads' attribute.
+      // 3. We will not show two ads in a row.
```

Comment:
how about formatting the code and comment in a more readable way:

```js
if (!this.isCurrentAdLoaded_ // 1. Ad must be loaded.
    || this.adsAreBlocked_(pageBeforeAd)  // 2. Pubs can opt out of ad placement using 'block-ads' 

```

Reply:
Done.